docs/advanced-features/event-driven-packaging.md

```diff
@@ -0,0 +1,181 @@
+<!-- markdownlint-disable -->
+# Event-Driven Packaging (EDP)
+
+> EDP is currently in private preview. Please contact Quilt
+directly to discuss pricing.
+
+## Overview
+
+Data can be added to your Amazon S3 buckets manually by internal
+team members and/or external collaborators, or automatically via
+instrumentation. No matter how it arrives into your buckets, it 
+is helpful, and sometimes critical, to be notified of this data 
+migration.
```

Comment:
Migration is not the right word. And the problem is really:
```suggestion
Data tend to be created in logical batches by machines, people, and pipelines.
Detecting these logical events from S3 events alone is complex and requires extensive custom logic.
```

Reply:
Applied